flaml/ml.py

```diff
@@ -265,14 +267,13 @@ def sklearn_metric_loss_score(
             used to calculate the metric. E.g., 2d for log_loss and 1d
             for others.
         y_true: A 1d numpy array of the true labels.
-        labels: A 1d numpy array of the unique labels.
+        label_list: A str list of the unique labels.

```

Comment:
Are you sure it's a list for non-NLP classification tasks?

Reply:
I am not sure which file to test for the non-nlp case, do you know?